src/app/authentication/sign-in/sign-in.component.html

```diff
@@ -14,9 +14,13 @@
          [(ngModel)]="user.password"
          required>
 
-  <button type="submit">Sign In</button>
+  <button appSubmit type="submit">Sign In</button>
 </form>
 
-<button type="button" (click)="logInWithGoogle()">Log In with Google</button>
+<button (click)="signInWithGoogle()">Sign In with Google</button>
+
 <button type="button" routerLink="../signup">Don't have an account?</button>
+<button type="button" routerLink="/dashboard">Kliknij sobie</button>
+<button type="button" (click)="signOut()">Log out</button>
```

Comment:
No need to specify type="button" for element outside of the form

Reply:
ok.